src/config/config.h

```diff
@@ -44,12 +45,19 @@ struct Version {
  */
 struct NrnUnitsLib {
     /// paths where nrnunits.lib can be found
-    static const std::vector<std::string> NRNUNITSLIB_PATH;
+    static std::vector<std::string> NRNUNITSLIB_PATH;
 
     /**
      * Return path of units database file
      */
     static std::string get_path() {
+        // first look for NMODLHOME env variable
+        if (const char* nmodl_home = std::getenv("NMODLHOME")) {
+            auto path = std::string(nmodl_home) + "/share/nrnunits.lib";
+            NRNUNITSLIB_PATH.insert(NRNUNITSLIB_PATH.begin(), path);
```

Comment:
could [emplace](https://en.cppreference.com/w/cpp/container/vector/emplace) be better?

Reply:
@cattabiani  : I can change that, didn't come to my mind as haven't used emplace too often :) 

But, does people replace `insert` with `emplace` for trivial types like string? I was reading [here](https://abseil.io/tips/112). As insert have `value_type&& val` argument, I assume it shouldn't matter.

Reply:
@pramodk: ` As insert have value_type&& val argument, I assume it shouldn't matter.`

Are you sure that's the overload that will be used?

Reply:
TBH, I am not sure and didn't check! but I have have changed insert to emplace already.